### PR TITLE
feat(settings) blur background when re-encrypting due to password change

### PR DESCRIPTION
**Pull request by stefandunca**
### Closes: #13992

| Design | Implementation |
| - | - |
| image | image |

### CODE REVIEW

File: ui/app/AppLayouts/Profile/popups/ConfirmChangePasswordModal.qml [hunk 1]
```diff
@@ -48,11 +48,59 @@ StatusDialog {
...
+
+            // Update the blur source only once, when the modal is shown for performance reasons
+            onVisibleChanged: {
+                if (!visible) {
+                    return;
+                }
+                blurSource.scheduleUpdate()
+            }
+
+            GaussianBlur {
```
Comment (caybro) [inline ui/app/AppLayouts/Profile/popups/ConfirmChangePasswordModal.qml:77]: `FastBlur` would be a better option, exactly for the performance reasons mentioned above :)

**Review by caybro — APPROVED** [review]
LGTM

Just the minor suggestion

File: ui/app/AppLayouts/Profile/popups/ConfirmChangePasswordModal.qml (hunk 1 above)
Comment (stefandunca) [reply ui/app/AppLayouts/Profile/popups/ConfirmChangePasswordModal.qml:77]: Thanks for looking into it. In tried `FastBlur` first but it doesn't fit the design (not so uniform blur). The alternative was to render only once the all window which is really negligible.

Comment (caybro) [reply ui/app/AppLayouts/Profile/popups/ConfirmChangePasswordModal.qml:77]: Yeah the FastBlur isn't as nice visually

Comment (stefandunca) [reply ui/app/AppLayouts/Profile/popups/ConfirmChangePasswordModal.qml:77]: image

**Pull request merged**